build.gradle

```diff
@@ -65,8 +75,18 @@ publishing {
 	}
 
 	// select the repositories you want to publish to
+	// to just publish to maven local, no extra repositories are necessary. Just use the task `publishToMavenLocal`.
 	repositories {
-		// uncomment to publish to the local maven
-		// mavenLocal()
+		// An example of a standard Nexus setup, for those wishing to publish their mod artifacts
+		// maven {
+		//	if (project.version.endsWith("-SNAPSHOT")) {
+		//		url = "https://nexus.myorganization.org/repository/maven-snapshots/"
+		//	} else {
+		//		url = "https://nexus.myorganization.org/repository/maven-releases/"
+		//	}
```

Comment:
I dont think this is really needed, its another thing to confuse new develeops. Very few people publish to maven, and if so there are good docs online? Possbly just link to https://docs.gradle.org/current/userguide/publishing_maven.html